### PR TITLE
Refurbish

### PR DESCRIPTION
**Pull request by mgumz**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,23 @@
-FROM golang:1.10-alpine as builder
-RUN apk --no-cache add curl git && curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-COPY . /go/src/github.com/travelping/ping-exporter
+FROM    golang:1.11-alpine3.8 as build-stage
+RUN     apk --no-cache --update add curl git dep
+COPY    . /go/src/github.com/travelping/ping-exporter
 WORKDIR /go/src/github.com/travelping/ping-exporter
-RUN dep ensure && CGO_ENABLED=0 GOOS=linux go build -a -o ping-exporter .
+RUN     dep ensure -v && \
+        CGO_ENABLED=0 GOOS=linux go build -v -a -o /ping-exporter .
```
Comment (grundrauschen) [inline Dockerfile:7]: Is there a reason, why a Makefile was created, but it is not used during the image build?

Comment (mgumz) [reply Dockerfile:7]: yeah: unclear how hub.docker.com picks up the build and how one can control what hub.docker.com shall use for building.

Comment (grundrauschen) [reply Dockerfile:7]: Well, if I am not wrong, the `RUN` statement runs inside your defined container. Therefore if you are able to run it locally, with installing make inside the container first, it should also run on Docker Hub, right?

Comment (mgumz) [reply Dockerfile:7]: the question: what gives? make only calls "docker build" as well. hows that different from hub.docker.com calling "docker build"?

**Pull request merged**